src/highlight.rs

```diff
@@ -88,57 +88,68 @@ fn token_style(kind: Kind, value: &str) -> Option<Style> {
     use crate::print::style::Style as S;
 
     match kind {
-        T::Keyword => Some(S::Keyword),
+        T::Keyword => {
+            let lc = value.to_lowercase();
+            if lc == "true" || lc == "false" {
```

Comment:
```suggestion
            if value.eq_ignore_ascii_case("true") || value.eq_ignore_ascii_case("false") {
```

`to_lowercase` is allocating, we try to ignore allocations if possible in hot paths.

Reply:
got it, thanks!